_includes/analytics.html

```diff
@@ -1,16 +1,15 @@
 {% if site.google_analytics %}
 	<!-- Google Analytics -->
+	<!-- Replace your google UA-XXXXXXXX-X tracking code  
+	Get a tracking code from Admin > Property > Tracking Info > Tracking Code
+	-->
+	<script async src="https://www.googletagmanager.com/gtag/js?id=UA-XXXXXXXX-X"></script>
 	<script>
-		(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-		(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-		m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-		})(window,document,'script','//www.google-analytics.com/analytics.js','ga');
+	  window.dataLayer = window.dataLayer || [];
+	  function gtag(){dataLayer.push(arguments);}
+	  gtag('js', new Date());
 
-		ga('create', '{{ site.google_analytics }}', 'auto');
-		ga('send', 'pageview', {
-		  'page': '{{ site.baseurl }}{{ page.url }}',
-		  'title': '{{ page.title | replace: "'", "\\'" }}'
-		});
+	  gtag('config', 'UA-XXXXXXXX-X');
```

Comment:
Use `{{ site.google_analytics }}` variable here, i.e.
```
<script async src="https://www.googletagmanager.com/gtag/js?id={{ site.google_analytics }}"></script>

// ...

gtag('config', '{{ site.google_analytics }}')
```

This way the user does not have to worry about directly editing the template source code.

Reply:
I will use {{ site.google_analytics }} great you told me about it 

I believe gtag is the new way to link the app with GA.  

I am not sure what is the difference between both method. The old method didn't work with me, tried many times, but gtag worked perfectly that is why I pushed it. 

In stackoverflow answer you shared, he mentioned about track features ..etc, it's not applicable to jekyll since it's just for personal blogging we really don't care to track parts of the app or features